src/core/errorHandler.js

```diff
@@ -10,6 +11,15 @@ function generateHandlerId({ name = '' } = {}) {
   return `${name}-${Math.random().toString(36).substr(2, 9)}`;
 }
 
+export const normalizeFileNameId = (filename) => {
+  let fileId = filename;
+  if (!fileId.startsWith('src')) {
```

Comment:
I just realized that nothing is obvious about what this code is doing. I would suggest adding a comment saying that on the server we get an absolute path and on the client we get a relative path. This function will make server and client code return the same value.

Reply:
ah sure, that totally makes sense.